.addon

```diff
@@ -275,6 +275,19 @@
     },
     "Summary": "",
     "Description": "",
-    "Public": false
+    "Public": false,
```

Comment:
This file has been deprecated I believe. You can remove the changes / delete it I think

Reply:
Removed this